Tempura/Sources/Navigation/Navigator.swift

```diff
@@ -302,9 +302,12 @@ public class Navigator {
 
     // if there is no route in common, ask the UIApplication to handle that
     if commonRouteIndex < 0 {
-      // swiftlint:disable:next force_unwrapping
-      let change = RouteChange.rootChange(from: old.first!.routeIdentifier, to: new.first!)
-      return [change]
+      if let fromIdentifier = old.first?.routeIdentifier, let toIdentifier = new.first {
```

Comment:
What was the use-case where `new.first` was `null`?

Reply:
The Firebase crash was from `old` being empty, but now I'm realizing I need to return a `show` route change. That just pushes the crash to somewhere else. I'm going to close and re-evaluate the fix here.